app/controllers/concerns/sortable.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Sortable
+  extend ActiveSupport::Concern
+
+  included do
+    helper_method :sort_column, :sort_direction
+  end
+
+  def order_by
+    "#{sort_column} #{sort_direction}"
+  end
+
+  def sort_column
+    return params[:column] if sortable_columns.include?(params[:column])
+    return "name"          if sortable_columns.include?("name")
+    "created_at"
+  end
+
```

Comment:
I would define a default implementation for `sortable_columns` with a note to override in including controllers.

```ruby
# Abstract method that should be overridden in including controllers
def sortable_columns
  []
end
```

Reply:
You could even raise `NotImplementedError` with the message.